.prettierignore

```diff
@@ -0,0 +1,22 @@
+
+target/
+node_modules/
+
+# Ignore files for PNPM, NPM and YARN
+pnpm-lock.yaml
+package-lock.json
+yarn.lock
+
+# global
+.github/
+
+# identity-wallet
```

Comment:
Such comments might be superfluous since the path explains what is ignored already, right?

Reply:
Unnecessary comments removed.